app/views/layouts/application.slim

```diff
@@ -127,22 +127,16 @@ html lang='en'
               => icon('fas', 'flag-checkered')
               | Return to race
         header#header.m-1.p-0 = yield(:header)
+        - if current_user && current_user.email.nil?
+          .alert.alert-danger
```

Comment:
I feel like the majority of users do not have a splits.io backed account, and this might be alarming with the current styling for something that was how the site was designed just a few weeks (months?) ago.

Reply:
This is actually not Splits.io-backed vs. non-Splits.io-backed, as when we added username/password sign in in #727 we migrated `TwitchUser` / `GoogleUser` emails to `User`. So this will only include users who had a null Twitch and Google email. AFAIK it's impossible to have a null Google email, so this was effectively only for users with null Twitch emails and no Google link. Although you have needed an email address to sign up for Twitch for years, the Twitch API will return a null email if you have not verified your email with them.

I checked the production database and about 14% of users (~6k) have null emails and will trigger this warning. I can still tone down this styling since that's not nothing.

Reply:
Oh, didn't realize this was covering a different use case.  It's less of a issue then in my mind, but that is still a large # of users so thanks for making that change.